controllers/falcon/falconnodesensor_controller.go

```diff
@@ -832,7 +832,7 @@ func (r *FalconNodeSensorReconciler) finalizeDaemonset(ctx context.Context, imag
 			// When the pods have a status of completed or running, increment the count.
 			// The reason running is an acceptable value is because the pods should be running the sleep command and have already cleaned up /opt/CrowdStrike
 			for _, pod := range pods.Items {
-				if pod.Status.Phase == "Completed" || pod.Status.Phase == "Running" {
+				if pod.Status.Phase == "Completed" || pod.Status.Phase == "Running" || pod.Status.Phase == "CrashLoopBackOff" {
```

Comment:
I have a feeling that my friend Gabe would tell me to use constants instead of hardcoded strings if I did this.